README.md

```diff
@@ -0,0 +1,59 @@
+# StoreFront
+
+[![license](https://img.shields.io/github/license/mashape/apistatus.svg?style=flat-square)](https://choosealicense.com/licenses/mit/)
+[![API Reference](https://img.shields.io/badge/API_reference-latest-blue.svg?style=flat-square)](https://groupby.github.io/storefront/index.html)
+
+StoreFront (SF) is an e-commerce UI component library that is fully integrated with the GroupBy ecosystem/services.
+
+## Getting Started
+
+### Prerequisites
+ - [yarn](https://yarnpkg.com/en/)
+
+## Setup
+Run the `./scripts/setup.sh` script from the root of the project monorepo in order to build all of the StoreFront packages.
+```sh
+  ./scripts/setup.sh
+```
+
+## Commands
+The following commands are in the context of an individual package contained within the StoreFront monorepo. The individual packages can be found within the [packages/@storefront](packages/@storefront) directory.
```

Comment:
Currently: "...commands are in...". Should be "...commands are run in...".

Reply:
Done, resolving this comment.